app/views/shared/_email_languages.html.erb

```diff
@@ -23,7 +23,7 @@ locals:
           locale == I18n.locale ?
             t('account.email_language.default', language: t("i18n.locale.#{locale}")) :
             t("i18n.locale.#{locale}"),
-          locale,
+          lang: locale,
           checked: selection ? selection.to_s == locale.to_s : I18n.locale.to_s == locale.to_s,
```

Comment:
I recall looking into this before and finding that the options for radio collection items are quite limiting and don't allow you to pass unsupported attributes, unfortunately ([related code](https://github.com/heartcombo/simple_form/blob/159b70aea62443aeb0e32382de3095bcb571ee0e/lib/simple_form/form_builder.rb#L381-L396)).

Maybe we can add a wrapper element to the text that's shown in the label and assign the `lang` to that?

```rb
content_tag(
  :span,
  locale == I18n.locale ?
    t('account.email_language.default', language: t("i18n.locale.#{locale}")) :
    t("i18n.locale.#{locale}"),
  lang: locale,
)
```

Reply:
@jmdembe What was the issue with the initial implementation where we added the the text element wrapper? I see the approach changed in recent commits.

Reply:
> @jmdembe What was the issue with the initial implementation where we added the the text element wrapper? I see the approach changed in recent commits.

First, I got into a tricky place with my commits, so it blew away the change that you suggested.
I tried the above method and I ran into the problem of not being able to select French as an option. I could not exactly pin down _why_ that was happening, so then I decided to take a different approach.

Reply:
Looking at [the original diff](https://github.com/18F/identity-idp/compare/4e72b02...6ffe396), I see that each collection item would have 3 values of the array, and the `locale` was being removed (line 26 of the removed code in the diff), perhaps unintentionally? IIRC, this represents the value of the radio input, so that might explain the issues you were seeing.

Reply:
ah, that was it. That was very unintentional.